src/test/scala/uk/gov/hmrc/perftests/itsass/EmploymentsRequests.scala

```diff
@@ -24,24 +24,109 @@ import uk.gov.hmrc.perftests.itsass.RequestsHelper._
 
 object EmploymentsRequests extends ServicesConfiguration {
 
-  def getEmploymentSummaryPage: HttpRequestBuilder = http("Get Employment Summary Page")
-    .get(s"$employmentsUrl/employment-summary")
+  def getEmploymentSummaryPage(taxYear: String): HttpRequestBuilder = http("Get Employment Summary Page")
+    .get(s"${employmentsUrl(taxYear)}/employment-summary")
     .check(status.is(200))
 
-  def getEmployerDetailsAndBenefitsPage(employmentId: String): HttpRequestBuilder = http("Get Employer Details And Benefits Page")
-    .get(s"$employmentsUrl/employer-details-and-benefits?employmentId=$employmentId")
-    .check(saveCsrfToken())
+  def getEOYEmploymentSummaryPage(taxYearEOY: String): HttpRequestBuilder = http("Get Employment Summary Page")
```

Comment:
I dont really think this method is needed, youve got the getEmploymentSummaryPage above which passes in a taxYear, so can just pass in taxYear-1 to it instead of needing this extra method

Reply:
Updated